Tribler/Core/Utilities/unicode.py

```diff
@@ -7,7 +7,18 @@
 
 import sys
 
-from six import text_type
+from six import binary_type, text_type
+
+
+def ensure_text(s, encoding='utf-8', errors='strict'):
```

Comment:
Could you make the encoding a non-optional field here?

My fear is that people start using this as a drop-in replacement for `unicode`, whereas this is only a replacement for the functionality of `unicode(string, encoding)` and not `unicode(obj)`. For instance:

```python
>>> unicode(1)
u'1'
>>> unicode(1, "UTF-8")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: coercing to Unicode: need string or buffer, int found
```

Reply:
Let's pick a different function name to reduce confusion...

__ensure_unicode()__ is like __six.ensure_text()__ except that encoding parameter is ___not___ optional

Reply:
Perfect 👍